utils/git_utils.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, path):
             self._repo = Repo(path)
         except (InvalidGitRepositoryError, NoSuchPathError):
             mkdir(path)
-            self._repo = Repo.init(path)
+            self._repo = Repo.init(path, kwargs="--initial-branch=main")
```

Comment:
Does this work ok? `kwargs` is normally pseudo syntax for a list of keyword arguments, https://stackoverflow.com/questions/64470052/how-to-set-default-branch-for-gitpython would suggest that `initial_branch='main'` would work